src/helm/benchmark/scenarios/code_scenario_apps_pinned_file_order.py

```diff
@@ -22,7 +22,7 @@ def get_apps_split_to_pinned_file_order(download_dir: str):
         file_path: str = os.path.join(download_dir, _APPS_PINNED_FILE_ORDER_FILENAME)
         ensure_file_downloaded(
             source_url=_APPS_PINNED_FILE_ORDER_URL,
-            download_dir=file_path,
+            target_path=file_path,
```

Comment:
Added quick fixes for typos in the call to ensure_file_downloaded. Not sure why mypy didn't catch this...